fuel/downloaders/base.py

```diff
@@ -55,6 +69,9 @@ def download(url, file_handle, chunk_size=1024):
         URL to download.
     file_handle : file
         Where to save the downloaded URL.
+    fd : open file handle, optional
```

Comment:
fd is a bit of a confusing name.  I'm also not sure about the choice of stderr. Seems like that be reserved for, well, errors, warnings, that sort of thing... @bartvm?